library/VTKExtensions/Rendering/vtkF3DRenderer.h

```diff
@@ -77,6 +77,13 @@ class vtkF3DRenderer : public vtkOpenGLRenderer
   void SetRaytracingSamples(int samples);
   ///@}
 
+  ///@{
```

Comment:
remove group doxygen doc

Reply:
Removed doxygen tags

Reply:
not correctly removed, Set/Get Use trackball should have them. SetUseOrthographicProjection shoult not.